src/appShell/App/usageAgreements/StudyAgreement.tsx

```diff
@@ -11,13 +11,9 @@ export function shouldShowStudyViewWarning() {
     // we want to show a warning message on private cbioportal instances
     // to prevent users from adding links in manuscripts
     // we don't want to show this in MSK CIS setting (iframe)
-    const showStudyViewWarning =
-        ['triage-portal', 'mskcc-portal'].includes(
-            AppConfig.serverConfig.app_name!
-        ) && !getBrowserWindow().isMSKCIS;
-
     return (
-        showStudyViewWarning &&
+        AppConfig.serverConfig.is_private_portal &&
+        !getBrowserWindow().isMSKCIS &&
```

Comment:
Maybe ok leave as it was, `is_private_portal` is pretty similar to using `app_name === mskcc-portal`. If we would want to add a prop for something that describes this it would maybe something like `show_mskcc_study_agreement_banner`. Just don't really see a use case for anyone outside of MSKCC and we would have to change the prop again if we'd generalize it, so maybe better to leave as it was